style.css

```diff
@@ -37,115 +41,120 @@ html {
 	padding: 15px;
 }
 
-.welcome-banner {
-	background-color: #EEEEEE;
-	height: 90vh;
-	
+@media screen and (max-width: 600px) {
+	.cv-button {
+	flex-direction: column;
+	font-size: 2rem;
+	}
 }
 
 .navigation-links {
 	color: #ffffff;
 	text-decoration: none;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
-    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #2866c9;
 }
 
-.hi-there-welcome-text {
-	display:flex;
-	align-items: center;
-	justify-content: center;
+@media screen and (max-width: 600px) {
+	.navigation-links {
+	font-size: 2rem;
+	}
+}
+
+/*navigation*/
+
+/*welcome*/
+
+.welcome-banner {
+	background-color: #EEEEEE;
+	height: 90vh;
+	flex-direction: column;
 	font-family: 'Varela Round', sans-serif;
-	font-size: 6rem;
-	padding-top: 10%;
 	color: #000000;
-	/*deciding whether to keep element*/
-/*    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #EFB7B7;*/
 }
 
-.welcome-text { 
+@media screen and (max-width: 600px) {
+	.welcome-banner {
+	flex-direction: column;
+	}
+}
+
+.small-title {
```

Comment:
Generally h1 should be the most important heading on the page. Consider using <span> tag around the second half of the sentence to change its size.